website/static/js/pages/registration-edit-page.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+require('css/registrations.css');
+
+var ko = require('knockout');
+var $ = require('jquery');
```

Comment:
Interesting!

It's being included over 100 times across the OSF, though - I'd prefer not to break with convention unless we plan to remove it from everywhere it's unnecessary.